src/UAEPass/README.md

```diff
@@ -0,0 +1,86 @@
+# UAEPass
+
+```bash
+composer require socialiteproviders/uaepass
+```
+# Set UAE PASS Environment
+```env
+UAEPASS_BASE_URL="https://stg-id.uaepass.ae"
+UAEPASS_CLIENT_ID="sandbox_stage"
+UAEPAS_CLIENT_SECRET="sandbox_stage"
+UAEPAS_REDIRECT_URI="http://my-app.test/callback"
+```
+
+## Installation & Basic Usage
+
+Please see the [Base Installation Guide](https://socialiteproviders.com/usage/), then follow the provider specific instructions below.
+
+### Add configuration to `config/services.php`
+
+```php
+'uaepass' => [
+    'base_url' => env('UAEPASS_BASE_URL'),
+    'client_id' => env('UAEPASS_CLIENT_ID'),
+    'client_secret' => env('UAEPAS_CLIENT_SECRET'),
+    'redirect' => env('UAEPAS_REDIRECT_URI')
```

Comment:
to use $this->getConfig('base_url') you need to add `base_url` here

Reply:
Please correct me if I'm wrong ? `base_url` is already added in env as `UAEPASS_BASE_URL`

Reply:
It needs to be added to the config in services.php or `getConfig` with the additional config keys wont find it

Reply:
Updated with default base_url in ReadMe https://github.com/SocialiteProviders/Providers/pull/1203/commits/3caf8cb6907d627096e8389f095ec73a130944cb

Reply:
I have tested and it's working fine.